test/browser/commands.test.js

```diff
@@ -19,18 +31,35 @@ describe('commands', () => {
 		});
 
 		it('should click at position', async() => {
-			const clickPos = { x: 0, y: 0 };
-			function onClick(e) {
-				clickPos.x = e.clientX;
-				clickPos.y = e.clientY;
-			}
 			window.addEventListener('click', onClick);
```

Comment:
Let's add the event listeners in a global `before` (just runs once) and remove them in a global `after`.